lib/utils.js

```diff
@@ -19,4 +20,35 @@ exports.md5 = function(str) {
 exports.url = function(url){
   if (~url.indexOf('://')) return url;
   return 'http://' + url;
-};
\ No newline at end of file
+};
+
+/**
+ * Delete all files in [dirPath] older than [ttl].  Does not modify subdirectories.
+ */
+
+exports.eraseOldFiles = function(dirPath, ttl) {
+  ttl = typeof ttl !== undefined ? ttl : 0;
+  var files = fs.readdirSync(dirPath);
+  files.forEach(function(file) {
+    var filePath = path.join(dirPath, file);
+    var stats = fs.statSync(filePath);
+    if (stats.isFile() && file.slice(0,10) === 'screenshot') {
+      var now = new Date().getTime();
```

Comment:
just calculate the 'expiration time outside the loop as now() - ttl
then check each mtime vs that.